utils/schema/event.js

```diff
@@ -0,0 +1,33 @@
+export default {
+  "@context": "https://schema.org",
+  "@type": "Event",
+  name: "اولین گرد همایی vue.js تهران",
+  startDate: "2019-09-21T14:00",
+  endDate: "2019-09-21T20:00",
+  location: {
+    "@type": "Place",
+    name: "Zavie Coworking Space",
+    address: {
+      "@type": "PostalAddress",
+      streetAddress: "Tehran Province, Tehran, Lashkari Expy",
+      addressLocality: "Snickertown",
+      postalCode: "19019",
+      addressRegion: "TH",
+      addressCountry: "IR"
+    }
+  },
+  image: ["https://vuejsconf.ir/img/poster2.jpg"],
+  description: "The First Vue.js Tehran Meetup",
+  offers: {
+    "@type": "Offer",
+    url: "https://www.example.com/event_offer/12345_201803180430",
```

Comment:
Is it a valid link?

Reply:
Sorry, this has been fixed in the new version where the venue was designated.